en/core-libraries/time.rst

```diff
@@ -69,45 +71,55 @@ as a UNIX timestamp.
 In test cases you can mock out ``now()`` using ``setTestNow()``::
 
     // Fixate time.
-    $now = new FrozenTime('2014-04-12 12:22:30');
-    FrozenTime::setTestNow($now);
+    $time = new FrozenTime('2021-01-31 22:11:30');
+    FrozenTime::setTestNow($time);
 
-    // Returns '2014-04-12 12:22:30'
-    $now = FrozenTime::now();
+    // Outputs '1/31/21, 10:11 PM'
+    echo FrozenTime::now();
 
-    // Returns '2014-04-12 12:22:30'
-    $now = FrozenTime::parse('now');
+    // Outputs '1/31/21, 10:11 PM'
+    echo FrozenTime::parse('now');
 
 Manipulation
 ============
 
-Once created, you can manipulate ``FrozenTime`` instances using setter methods::
+Once created, you cannot manipulate ``FrozenTime`` instances as it is immutable. Using setter methods will not work::
```

Comment:
Paragraph shouldn't start with _you cannot_.

Start with correct example which show you need to assign new instance to variable.

You can insert incorrect example later in paragraph.

Reply:
This is misleading. It sounds like you can't call the methods.